articles/data-factory/data-flow-expression-functions.md

```diff
@@ -16,658 +16,703 @@ ms.date: 02/15/2019
 ### <code>covariancePopulation</code>
 <code><b>covariancePopulation(<i>&lt;value1&gt;</i> : number, <i>&lt;value2&gt;</i> : number) => double</b></code><br/><br/>
 Gets the population covariance between two columns
-* ``covariancePopulation(sales, profit) -> 122.12``
----
+* ``covariancePopulation(sales, profit) -> ``
+*********************************
 ### <code>covariancePopulationIf</code>
 <code><b>covariancePopulationIf(<i>&lt;value1&gt;</i> : boolean, <i>&lt;value2&gt;</i> : number, <i>&lt;value3&gt;</i> : number) => double</b></code><br/><br/>
 Based on a criteria, gets the population covariance of two columns
-* ``covariancePopulationIf(region == 'West', sales) -> 122.12``
----
+* ``covariancePopulationIf(region == 'West', sales) -> ``
+*********************************
 ### <code>covarianceSample</code>
 <code><b>covarianceSample(<i>&lt;value1&gt;</i> : number, <i>&lt;value2&gt;</i> : number) => double</b></code><br/><br/>
 Gets the sample covariance of two columns
-* ``covarianceSample(sales, profit) -> 122.12``
----
+* ``covarianceSample(sales, profit) -> ``
+*********************************
 ### <code>covarianceSampleIf</code>
 <code><b>covarianceSampleIf(<i>&lt;value1&gt;</i> : boolean, <i>&lt;value2&gt;</i> : number, <i>&lt;value3&gt;</i> : number) => double</b></code><br/><br/>
 Based on a criteria, gets the sample covariance of two columns
-* ``covarianceSampleIf(region == 'West', sales, profit) -> 122.12``
----
+* ``covarianceSampleIf(region == 'West', sales, profit) -> ``
+*********************************
 ### <code>crc32</code>
 <code><b>crc32(<i>&lt;value1&gt;</i> : any, ...) => long</b></code><br/><br/>
 Calculates the CRC32 hash of set of column of varying primitive datatypes given a bit length which can only be of values 0(256), 224, 256, 384, 512. It can be used to calculate a fingerprint for a row
-* ``crc32(256, 'gunchus', 8.2, 'bojjus', true, toDate('2010-4-4')) -> 3630253689``
----
+* ``crc32(256, 'gunchus', 8.2, 'bojjus', true, toDate('2010-4-4')) -> 3630253689L``
+*********************************
 ### <code>cumeDist</code>
 <code><b>cumeDist() => integer</b></code><br/><br/>
 The CumeDist function computes the position of a value relative to all values in the partition. The result is the number of rows preceding or equal to the current row in the ordering of the partition divided by the total number of rows in the window partition. Any tie values in the  ordering will evaluate to the same position.
-* ``cumeDist() -> 1``
----
+* ``cumeDist() -> ``
+*********************************
 ### <code>currentDate</code>
 <code><b>currentDate([<i>&lt;value1&gt;</i> : string]) => date</b></code><br/><br/>
 Gets the current date when this job starts to run. You can pass an optional timezone in the form of 'GMT', 'PST', 'UTC', 'America/Cayman'. The local timezone is used as the default.
-* ``currentDate() -> 12-12-2030``
-* ``currentDate('PST') -> 12-31-2050``
----
+* ``currentDate() == toDate('2250-12-31') -> false``
```

Comment:
I don't think this is clear on what currentDate returns

Reply:
Kiran changed example, think it's clearer than before